src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js

```diff
@@ -84,7 +84,7 @@ export function isValidPTFTable() {
     }
     else {
          const errorMsg = `The uploaded table is not valid. The PTF option requires pid.
-                        Please select one of the other upload options`;
+                        Please select the "Other" upload option for tables that do not meet these requirements.`;
```

Comment:
I think we want to keep the other options as a text and not indicating the option as alternative. The previous text was good, no need to change unless i'm missing something else.
Same for WISE case below.

Reply:
See IRSA-782 for the exact text.

Reply:
Okay, now I am confused.  The exact text in the ticket is what I put here, which is not what they want.  What they want is what we have already.  I'll just revert the change.